code/__DEFINES/colors.dm

```diff
@@ -50,14 +50,15 @@
 #define COLOR_NAVY						"#000080"
 #define COLOR_BLUE_GRAY					"#75A2BB"
 
-#define COLOR_PINK						"#FFC0CB"
-#define COLOR_LIGHT_PINK				"#ff3cc8"
-#define COLOR_MOSTLY_PURE_PINK			"#E4005B"
-#define COLOR_MAGENTA					"#FF00FF"
-#define COLOR_STRONG_MAGENTA			"#B800B8"
-#define COLOR_PURPLE					"#800080"
-#define COLOR_VIOLET					"#B900F7"
-#define COLOR_STRONG_VIOLET				"#6927c5"
+#define COLOR_PINK "#FFC0CB"
+#define COLOR_LIGHT_PINK "#ff3cc8"
+#define COLOR_MOSTLY_PURE_PINK "#E4005B"
+#define COLOR_BLUSH_PINK "#DE5D83"
+#define COLOR_MAGENTA "#FF00FF"
+#define COLOR_STRONG_MAGENTA "#B800B8"
+#define COLOR_PURPLE "#800080"
+#define COLOR_VIOLET "#B900F7"
+#define COLOR_STRONG_VIOLET "#6927c5"
```

Comment:
Would like these to get their unified indentation back (Where they're all on the same column)